### ISSUE TITLE
feature request: set new server

### ISSUE THREAD

**Issue by somedevreally** [dejablue79/pfsense_VPN_optimizer#3]
Hello,

can we have a option to set different region server?
say am connected to UK server, and i want to change/set to NL server
something like `/set=nlxx.nordvpn.com`
not sure if its possible with multi-vpn setup

thanks again

**Comment by dejablue79** [comment]
I think it will be possible, ill update you.

**Comment by dejablue79** [comment]
If you are willing to try, I have add something like this:
http://localhost:5000/replace/nordvpn?loc=de&with=nl
basically: /replace/<provider>?loc=<old location>&with=<new location>

**Comment by somedevreally** [comment]
it says page not found

**Comment by dejablue79** [comment]
Did you pull and rebuild the docket image?

**Comment by somedevreally** [comment]
yes let me try again

**Comment by somedevreally** [comment]
works perfectly, you are awesome

**Comment by somedevreally** [comment]
this is so cool, we need some love for PIA too, just to change regions?

**Comment by dejablue79** [comment]
PIA is not following the two letters country code but actually the name of the country and the regions.
I guess it is possible but I rather focus on Nord and Proton for now
